compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Check.hs

```diff
@@ -129,6 +129,16 @@ kindOfBuiltin = \case
   BTAny -> KStar
   BTTypeRep -> KStar
 
+checkKind :: MonadGamma m => Kind -> m ()
```

Comment:
Shouldn't we run this on `ETyLam`, `DefTypeSyn` and `DefDataType` as well?

Reply:
Feeling very silly because of course we should. I added these.

Reply:
Thanks for the kind words :-)